python/scripts/bash/start-dev.sh

```
@@ -0,0 +1,2 @@
+python setup.py
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)